plugins/jobtype/src/azkaban/jobtype/HadoopSparkJob.java

```diff
@@ -53,6 +53,37 @@
  * The Azkaban adaptor for running a Spark Submit job.
  * Use this in conjunction with  {@link azkaban.jobtype.HadoopSecureSparkWrapper}
  *
+ * This class is used by azkaban executor to build the classpath, main args, env and jvm props
```

Comment:
Nice comment.

What do you think about putting this comment in a sample plugin.properties with the configs commented out?  Or put the comment in the constant file I suggested earlier?